src/internal/serviceenv/config.go

```diff
@@ -115,6 +115,7 @@ type EnterpriseSpecificConfiguration struct {
 	ActivateAuth            bool   `env:"ACTIVATE_AUTH,default=false"`
 	AuthRootToken           string `env:"AUTH_ROOT_TOKEN,default="`
 	AuthConfig              string `env:"AUTH_CONFIG,default="`
+	AuthClientSecret        string `env:"AUTH_CLIENT_SECRET="`
```

Comment:
might need to be `,default=` instead of ending in `=`?

Reply:
yikes, thx